gef-ui/server/services/api.js

```diff
@@ -400,6 +400,37 @@ const upsertAmendment = async ({ facilityId, dealId, amendment, userToken }) =>
   }
 };
 
+/**
+ * @param {Object} param
+ * @param {string} param.facilityId
+ * @param {string} param.amendmentId
+ * @param {import('@ukef/dtfs2-common').PortalFacilityAmendmentUserValues} param.update
+ * @param {string} param.userToken
+ * @returns {Promise<(import('@ukef/dtfs2-common').PortalFacilityAmendmentWithUkefId)>}>}
+ */
+const updateAmendment = async ({ facilityId, amendmentId, update, userToken }) => {
+  if (!isValidMongoId(facilityId)) {
+    console.error('Invalid facility ID %s', facilityId);
+    throw new InvalidFacilityIdError(facilityId);
+  }
+
+  if (!isValidMongoId(amendmentId)) {
+    throw new Error('Invalid amendment ID');
+  }
+
+  const payload = {
+    update,
+  };
+
+  try {
+    const { data } = await Axios.patch(`/gef/facilities/${facilityId}/amendments/${amendmentId}`, payload, { ...config(userToken) });
+    return data;
+  } catch (error) {
```

Comment:
Do we want to return the patch response? (and maybe validate it with the schema)
```suggestion
    return await Axios.patch(`/gef/facilities/${facilityId}/amendments/${amendmentId}`, payload, { ...config(userToken) });
```